src/eventable.js

```diff
@@ -82,11 +83,13 @@ function getEventableModule (notifyContext) {
       }
 
       const eventListeners = listeners[event]
-      if (eventListeners === undefined) return
+      if (!Array.isArray(eventListeners)) return
```

Comment:
Not sure about this more explicit test. If eventListeners is another type this will now silently return. I think it would be better if it threw an exception as any other type than undefined would be an error in `eventable`.